install_config/install/advanced_install.adoc

```diff
@@ -294,6 +294,53 @@ allocate 10.128.0.0/23, 10.128.2.0/23, 10.128.4.0/23, and so on. This *cannot* b
 re-configured after deployment.
 |===
 
+[[configuring-console-customization]]
+
+*Configuring for console customization*
+
+The following table describes the configurations to be done in the master config for console customization:
+
+
+.Console Customization Variables
+[options="header"]
+|===
+
+|Type |Variable |Value
```

Comment:
@adellape should this section mirror the other variable sections, having "Variable" and "Purpose" columns? The "Purpose" column could then add a description for each variable and provide an example or default value.